data/Nigeria/Senate/sources/instructions.json

```diff
@@ -0,0 +1,26 @@
+{
+  "sources": [
+    {
+      "file": "morph/data.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "everypolitician-scrapers/nigeria-national-assembly",
+        "query": "SELECT * FROM data WHERE js_position = 'Sen' AND area != ''"
```

Comment:
FYI: I don't think the cast to int is actually necessary — the important part is just that the order is consistent over time. But there's certainly no harm in doing it.